modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -29,7 +29,8 @@ endif::[]
 [id="installation-adding-registry-pull-secret_{context}"]
 = Configuring credentials that allow images to be mirrored
```

Comment:
The step order is off in this [link](https://102851--ocpdocs-pr.netlify.app/openshift-rosa-hcp/latest/openshift_images/samples-operator-alt-registry.html#installation-adding-registry-pull-secret_samples-operator-alt-registry): 

<img width="1460" height="1401" alt="Screenshot From 2025-11-21 15-53-56" src="https://github.com/user-attachments/assets/13f34406-d545-45d3-b6bd-dac37da15c24" />

Reply:
oh yeah, good catch